doc/api/sqlite.md

```diff
@@ -429,6 +429,18 @@ exception.
 | `TEXT`    | {string}             |
 | `BLOB`    | {Uint8Array}         |
 
+## `node:sqlite` module methods and properties
+
+### `sqlite.constants`
```

Comment:
I think this should be combined with the section on constants immediately below.

Reply:
```
## SQLite constants

### `sqlite.constants`

<!-- YAML
added: REPLACEME
-->

* {Object}

An object containing commonly used constants for SQLite operations.
```

Something like this maybe?

Reply:
Yes, but take a look at how https://nodejs.org/api/fs.html#fs-constants is structured.

Reply:
Silly question, @cjihrig . How can I see docs locally? I performed the changes. I'd like to see how it looks - and send a screenshot - before pushing the commit.

Reply:
You can see the generated HTML in `out/doc/api/sqlite.html`. I think `make doc` generates the files.

Reply:
What do you think?

<img width="531" alt="image" src="https://github.com/user-attachments/assets/bced99e4-b2ad-42b6-b633-9170d1d54461">

<img width="911" alt="image" src="https://github.com/user-attachments/assets/4de6c4a0-f44b-4616-a0a7-9b19cf9dc05f">

Reply:
LGTM